src/frontend/Player/createPlyrPlayer.ts

```diff
@@ -59,6 +59,16 @@ export const createPlyrPlayer = (
       }
     }
     xapiStatement.initialized(contextExtensions);
+
+    const playLarge: Nullable<HTMLButtonElement> = document.querySelector(
```

Comment:
I understand this works in our present circumstances because we're in an iframe which will probably only ever contain one plyr player.

Still, IMO you should use the component ref to interact with the DOM, maybe by changing the signature for `createPlayer` to get the ref for `VideoPlayer` and querying in that for `<video>` and for the `.plyr__control--overlaid`

Reply:
I can access the element directly in the Plyr instance.